entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/vocabulary/LdProfiles.java

```diff
@@ -0,0 +1,69 @@
+package eu.europeana.entitymanagement.definitions.model.vocabulary;
+
+import eu.europeana.entitymanagement.definitions.exceptions.ConceptSchemeProfileValidationException;
+
+/**
+ * This enumeration is intended for Linked Data profiles
+ * 
+ * @author GrafR
+ *
+ */
+public enum LdProfiles implements ProfileKeyword {
```

Comment:
please remove this enumeration, we have different specifications, and the Profiles are defined now in EntityProfile enumeration

Reply:
Removed.